orchagent/portsorch.cpp

```diff
@@ -2515,6 +2534,16 @@ void PortsOrch::doPortTask(Consumer &consumer)
             }
             else
             {
+                /* Skip configuring recirc port for now because the current SAI implementation of some vendors
```

Comment:
why getPort() will fail ? What does we need from SAI ?

Reply:
The SAI implementation we tested has limited support for recirc port.  For example, the only supported port attribute in SAI get call is lane set for a recirc port.  This is why we skipped port setting for recirc port for now.